root/static/scripts/timeline.js

```diff
@@ -583,7 +620,11 @@ class TimelineLine {
                 } else if (graph === 'overview') {
                     options.series = { lines: { lineWidth: 1 }, shadowSize: 0 };
                     options.xaxis = { mode: "time", minTickSize: [1, "year"] };
-                    options.yaxis = { tickFormatter: function () { return '' } };
+                    options.yaxis = {
+                        tickFormatter: function () {
+                            return '';
+                        }
+                    };
```

Comment:
This was changed to `{ tickFormatter: () => '' };` in the master branch via another PR, so can be reverted.